### DISCUSSION TITLE
Docs don't mention best practices for testing serverless functions

### DISCUSSION THREAD

**Discussion by jakeleventhal** [vercel/community#2363] [Idea]
I can't seem to find anything that mentions the best practices for writing tests for serverless functions. It seems easy to accidentally ship code that runs on jest unit tests but doesn't work in the wild due to runtime discrepancies. I saw there is the vercel jest edge runtime package, but there aren't many great examples provided to ensure consistency. Same goes for the Vercel node runtime.

**Comment by amyegan** [comment]
Thanks for making this suggestion! I'm moving it to the "Ideas" category so it can be considered for future addition to the docs
